services/server/server.go

```diff
@@ -102,6 +104,7 @@ func NewServer(params *ServerParams) (*Server, error) {
 		prometheus:         params.Prometheus,
 		supervisord:        params.Supervisord,
 		telemetryService:   params.TelemetryService,
+		checksService:      params.ChecksService,
```

Comment:
But we don't use it, right? Then why pass it at all?

Reply:
Nope, It's not used yet, but I thought we will finish checks in single PR.

Reply:
I think we should finish SAAS-65 today in that PR, and then start a next one

Reply:
Checks service disabled, PR renamed